tailwind.base.config.js

```diff
@@ -19,17 +19,18 @@ module.exports = {
         'secondary-darker': 'var(--color-secondary-darker)',
         'secondary-darkest': 'var(--color-secondary-darkest)',
         main: 'var(--color-main)',
+        title: 'black',
```

Comment:
We shouldn't need a new color here, as it does not really fit with our concept of theme. A theme is 4 colors: secondary, primary, main (text) and background. Titles have their own font but their color should be picked from the available palette offered by thos 4 colors.

Reply:
Discussing with @fgravin we thought it would be handy to be able to change all the title colors (at least from the tailwind config). Unfortunately, `black` cannot not be inferred from the 4 theme colors and using `text-black` in the templates, would make a change less flexible.

Reply:
Yes `--color-main-900` is not dark enough, so if we put `text-black` it's ok but it means we can't change it.
If we have a class for this specfic elements, it means we can imagine to customize it in the future (via TW config or via css variable or configuration if needed).
Not a big deal for now anyway.

Reply:
I honestly don't see the point of this style instead of `title-black` since it's not customizable any more.

Another solution would be to have `main-light`, `main-lighter`, `main-dark` and `main-darker` classes. But honestly this should not be needed because `main` was originally intended to be very close to black.